tests/EventTest.php

```diff
@@ -165,36 +160,25 @@ public function testToArrayWithBreadcrumbs(): void
         $this->assertSame($breadcrumbs, $data['breadcrumbs']['values']);
     }
 
-    public function testToArrayWithMessageWithFormatted(): void
+    /**
+     * @dataProvider getMessageDataProvider
+     */
+    public function testGetMessage(array $message, array $expectedValue): void
     {
-        $expected = [
-            'message' => 'foo @bar',
-            'params' => ['@bar' => 'bar'],
-            'formatted' => 'foo bar',
-        ];
-
         $event = new Event();
-        $event->setMessage('foo @bar', ['@bar' => 'bar'], 'foo bar');
+        call_user_func_array([$event, 'setMessage'], $message);
 
-        $data = $event->toArray();
-
-        $this->assertArrayHasKey('message', $data);
-        $this->assertSame($expected, $data['message']);
+        $this->assertSame($expectedValue['message'], $event->getMessage());
+        $this->assertSame($expectedValue['params'], $event->getMessageParams());
+        $this->assertSame($expectedValue['formatted'], $event->getMessageFormatted());
     }
 
-    public function testGetMessage(): void
+    public function getMessageDataProvider(): array
     {
-        $event = new Event();
-        $event->setMessage('foo @bar', ['@bar' => 'bar'], 'foo bar');
-        $this->assertSame('foo @bar', $event->getMessage());
-        $this->assertSame(['@bar' => 'bar'], $event->getMessageParams());
-        $this->assertSame('foo bar', $event->getMessageFormatted());
-
-        $event = new Event();
-        $event->setMessage('foo %s', ['bar']);
-        $this->assertSame('foo %s', $event->getMessage());
-        $this->assertSame(['bar'], $event->getMessageParams());
-        $this->assertNull($event->getMessageFormatted());
+        return [
+            [['foo %s', ['bar']], ['message' => 'foo %s', 'params' => ['bar'], 'formatted' => NULL]],
```

Comment:
Please split this array into multiple lines (have a look at how other data providers are formatted)